.github/workflows/js.yml

```diff
@@ -1,14 +1,28 @@
 name: JS tests
 on: [push, pull_request]
 jobs:
-  js-linter:
-    name: JS linter
```

Comment:
The previous name was okay (BTW please start your command names with a capital letter)